chemprop/cli/train.py

```diff
@@ -842,7 +864,34 @@ def train_model(
 
         torch.manual_seed(seed)
 
-        model = build_model(args, train_loader.dataset, output_transform, input_transforms)
+        if args.checkpoint or args.model_frzn is not None:
+            mpnn_cls = (
+                MulticomponentMPNN
+                if isinstance(train_loader.dataset, MulticomponentDataset)
+                else MPNN
+            )
+            model_path = model_paths[model_idx] if args.checkpoint else args.model_frzn
+            model = mpnn_cls.load_from_file(model_path)
+
+            if args.checkpoint:
+                model.apply(
+                    lambda m: setattr(m, "p", args.dropout)
+                    if isinstance(m, torch.nn.Dropout)
+                    else None
+                )
```

Comment:
Why don't we use the dropout value that is in the loaded model file?

Reply:
For model fine-tuning, I think it makes sense to adjust the dropout rate to suit the new task or dataset, as the previous rate was optimized for the pre-trained dataset and may not be suitable for the new dataset.